docs/site/Using-strong-error-handler.md

```diff
@@ -0,0 +1,234 @@
+---
+title: 'Using strong-error-handler'
+lang: en
+keywords: LoopBack 4, error handling
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-strong-error-handler.html
+---
+
+# strong-error-handler
+
+This package is an error handler for use in both development (debug) and
+production environments.
+
+In LoopBack 4, the request handling process starts with the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html), a simple class with
+five injected helper methods in the constructor. It is the gatekeeper of all
+requests to the app. Errors are handled by one of the Sequence actions,
+[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
+package to send back an HTTP response describing the error.
+
+LB4 apps require 3.x versions of `strong-error-handler`.
+
+## Installation
+
+```bash
+$ npm install --save strong-error-handler
+```
+
+## Usage
+
+As mentioned above, errors are handled by the `reject` action of the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html). By default,
+implementation of `reject`, `strong-error-handler` omits details from error
+responses to prevent leaking sensitive information.
+
+`strong-error-handler` also can return full error stack traces and internal
+details of any error objects to the client in the HTTP responses once the
+`debug` flag is set. Please check
+[Sequence - Handling errors](Sequence.md#handling-errors) section for usages and
+examples.
+
+In general, `strong-error-handler` must be the last middleware function to be
+registered.
+
+The above configuration will log errors to the server console, but not return
+stack traces in HTTP responses. For details on configuration options, see below.
+
+### Working with Express middleware
+
+Under the hood, LoopBack leverages [Express](https://expressjs.com) framework
+and its concept of middleware. To avoid common pitfalls, it is not possible to
+mount Express middleware directly on a LoopBack application.
```

Comment:
This has changed since the introduction of Express middleware support in `@loopback/rest@4.0.0`. But the key is that the error handler is stilled used.